plugins/modules/zos_operator.py

```diff
@@ -130,38 +170,161 @@ def run_module():
         rc_message = run_operator_command(new_params)
         result["rc"] = rc_message.get("rc")
         result["content"] = rc_message.get("message").split("\n")
+        if result["rc"] == 0:
+            if len(result["content"]) > 3:
```

Comment:
@richp405 I don't know the answer and positional values make me nervous so my question is, what is special about length 3? Are we certain index 2 will always be populated (see line 175). Just asking for clarity, this is a good place for a comment for others to understand who might work on the code.

Reply:
@blakeinate thanks for reminding in python 3 int's are unbounded :) , so @richp405 we can consider this comment moot.

Reply:
len(result of split) = #/lines split.  The line just means "if there are at least 4 lines of output".  I can add that as inline comment.

Reply:
Looking back at the code, I'm realizing my thinking, that invalid or error always appeared on the third line...the test was to make sure there wasn't a bounds error.  Blake found an edge-case I hadn't considered, so the change I'll propose is to return the 3 pieces separately, then check the first few lines of stderr and stdout with mini-comprehensions, for the trigger text.